### PR TITLE
Display catpcha for crypsis on quick reply for Joomla! 3.4+

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

**Comment by 810** [comment]
Only showed on first message, on every message still not working.

We need to find out how to do it.

**Comment by xillibit** [comment]
Now i have this error message in javascript console : "Error: ReCAPTCHA placeholder element must be empty"

File: components/com_kunena/site/controller/topic/item/message/display.php [hunk 1]
```diff
@@ -64,15 +64,15 @@ protected function before()
...
 				$captcha_pubkey = $params->get('public_key');
 				$catcha_privkey = $params->get('private_key');
-				
+
 				if (!empty($captcha_pubkey) && !empty($catcha_privkey))
 				{
 					JPluginHelper::importPlugin('captcha');
 					$dispatcher = JDispatcher::getInstance();
-					$result = $dispatcher->trigger('onInit', 'dynamic_recaptcha_1');
+					$result = $dispatcher->trigger('onInit', 'kreply' . $this->message->displayField('id') . '_form');
```
Comment (810) [inline components/com_kunena/site/controller/topic/item/message/display.php]: change to:

````
$result = $dispatcher->trigger('onInit', "dynamic_recaptcha_{$this->message->id}");
````

File: components/com_kunena/site/template/crypsis/layouts/message/edit/quickreply.php [hunk 2]
```diff
@@ -93,7 +93,7 @@
 		<?php if (!empty($this->captchaEnabled)) : ?>
 			<div class="control-group">
 				<label class="control-label"><?php echo JText::_('COM_KUNENA_CAPDESC'); ?></label>
-					<div class="controls"> <div id="dynamic_recaptcha_1"> </div> </div>
+					<div class="controls"> <div id="kreply<?php echo $message->displayField('id'); ?>_form"> </div> </div>
```
Comment (810) [inline components/com_kunena/site/template/crypsis/layouts/message/edit/quickreply.php]: change to:
````
<div class="controls"><div id="dynamic_recaptcha_<?php echo $this->message->id; ?>"> </div></div>
````

**Comment by 810** [comment]
Cool got it working, now we can support full recaptcha everywhere

**Comment by 810** [comment]
please also add on general.less
````
form#postform div.control-group {
	margin-left: 10px
}
````

**Pull request merged**